symplyphysics/laws/hydro/static_pressure_from_height.py

```diff
@@ -0,0 +1,32 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input, validate_output)
+
+# Description
+## Every liquid has pressure due to its own weight, also known as static pressure. Static and dynamic pressure added together form total pressure of the liquid.
+## Law: p = ro * g * h, where
+## ro is density of liquid,
+## g is gravity at the surface of liquid
+## h is height of liquid column or depth within a substance
+
+# Conditions
+## Constant density throughout the liquid
+## No variation of g, because height of liquid column is often reasonably small compared to the radius of the Earth
+
+liquid_density = Symbol("liquid_density", units.mass / units.volume)
+gravitational_acceleration = Symbol("gravity", units.meter / units.second**2) #normal gravity at the equator approx. = 9.7803267715 m/s^2 
```

Comment:
This constant is defined somewhere in SymPy. You can look for it in SymPy library or check through the symplyphysics code for another example of this constant usage.